migrations/0.0.0.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Module dependencies
+ */
+
+var async = require('async')
+var semver = require('semver')
+var Role = require('../models/Role')
+var Scope = require('../models/Scope')
+
+/**
+ * Data
+ */
+
+var defaults = {
+  roles: [
+    { name: 'authority' },
+    { name: 'developer' }
+  ],
+
+  scopes: [
+    { name: 'openid', description: 'View your identity' },
+    { name: 'profile', description: 'View your basic account info' },
+    { name: 'client', description: 'Register and configure clients' },
+    { name: 'realm', description: 'Configure the security realm' }
+  ],
+
+  permissions: [
+    ['authority', 'realm'],
+    ['developer', 'client']
+  ]
+
+}
+
+/**
+ * Insert Roles
+ */
+
+function insertRoles (done) {
+  async.map(defaults.roles, function (role, callback) {
+    Role.insert(role, function (err, instance) {
+      callback(err, instance)
+    })
+  }, function (err, roles) {
+    done(err, roles)
+  })
+}
+
+/**
+ * Insert Scopes
+ */
+
+function insertScopes (done) {
+  async.map(defaults.scopes, function (scope, callback) {
+    Scope.insert(scope, function (err, instance) {
+      callback(err, instance)
+    })
+  }, function (err, scopes) {
+    done(err, scopes)
+  })
+}
+
+/**
+ * Assign Permissions
+ */
+
+function assignPermissions (done) {
+  async.map(defaults.permissions, function (pair, callback) {
+    Role.addScopes(pair[0], pair[1], function (err, result) {
+      callback(err, result)
+    })
+  }, function (err, results) {
+    done(err, results)
+  })
+}
+
+/**
+ * Exports
+ */
+
+module.exports = function (version) {
+  return function migration_0_0_0 (next) {
+    if (semver.satisfies(version, '<=0.0.0')) {
```

Comment:
This is different behaviour than what we currently have. With this check, inserting roles, scopes, and assigning permissions only happens if there is no existing data. Prior, this would all get run regardless of the version of the data in the database, to alleviate concerns of users moving up from earlier versions where this logic was run solely by `nv migrate`.

It looks like this should either a) always run, which also helps in the case an admin accidentally deletes any of these entries from the database, or b) be a migration defined for the latest version of Connect that did not yet run these scripts automatically at boot-time.

I'm vouching for "a".